lib/v2/12306/index.js

```diff
@@ -0,0 +1,102 @@
+const got = require('@/utils/got');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+const rootUrl = 'https://kyfw.12306.cn';
+
+async function getJSESSIONID(linkUrl) {
+    const res = await got({
+        method: 'get',
+        url: linkUrl,
+        headers: {
+            UserAgent: 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.132 Safari/537.36',
```

Comment:
Will it work with RSSHub's default UA?
https://github.com/DIYgod/RSSHub/blob/471ea1a848770cee4ae655e3e34467c734c36d94/lib/config.js#L40